source.txt

```diff
@@ -506,6 +506,11 @@ Table of Contents
     SHOULD connect to the given provider account, as defined in Section
     10.
 
+    If the `remotestorage` field exists in the URL fragment, the application
+    SHOULD ignore any other parameters such as `access_token` or `state`, to
+    ensure compatibility with servers that implement older versions of this
+    specification.
+
```

Comment:
I was thinking about this for a bit, and it's actually not clear to me how ignoring parameters ensures compatibility with older servers. I think it's the other way around: requiring/expecting them would break compatibility with *newer* servers, no?